SPECS/mariner-release/mariner-release.spec

```diff
@@ -62,7 +62,7 @@ EOF
 %config(noreplace) %{_sysconfdir}/issue.net
 
 %changelog
-* Thu Feb 01 2023 Mykhailo Bykhovtsev <mbykhovtsev@microsoft.com> - 3.0-3
+* Thu Feb 01 2024 Mykhailo Bykhovtsev <mbykhovtsev@microsoft.com> - 3.0-3
```

Comment:
Can you split this file kntk a different PR?

Reply:
Removed from this one